### PR TITLE
image-viewer-multiple-images

### PR DESCRIPTION
**Pull request by MichaelNickolas**
to add multiple images to a card via URL

### CODE REVIEW

File: help/en/docs/widget-custom.md [hunk 1]
```diff
@@ -572,6 +572,18 @@ To copy the URL for an image, right click on the photo then 'Copy image address'
 
 *![image-viewer-save-image](images/widget-custom/image-viewer-save-image.png)*
 
+Additionally, you can add multiple images for a specific entry. First, head to the 🌳 All National Park Sites Card and click into the Park Image field.
```
Comment (nataliemisasi) [inline help/en/docs/widget-custom.md:575]: Make the language more generic (no specific mention of template / column name)
```suggestion
Additionally, you can add multiple images for a specific record by adding multiple image URLs, separated by a space or new line, into a single cell.
```

File: help/en/docs/widget-custom.md [hunk 2]
```diff
@@ -572,6 +572,18 @@ To copy the URL for an image, right click on the photo then 'Copy image address'
 
 *![image-viewer-save-image](images/widget-custom/image-viewer-save-image.png)*
 
+Additionally, you can add multiple images for a specific entry. First, head to the 🌳 All National Park Sites Card and click into the Park Image field. 
+
+*![image-viewer-park-card](images/widget-custom/image-viewer-park-card.png)*
```
Comment (nataliemisasi) [inline help/en/docs/widget-custom.md:577]: delete this screenshot - just show the one with two links, separated by newline

File: help/en/docs/widget-custom.md [hunk 3]
```diff
@@ -572,6 +572,18 @@ To copy the URL for an image, right click on the photo then 'Copy image address'
...
+
+Here, you can paste an additional link to add another photo. For proper spacing, use a space or a new line. A comma will not work to seperate the links.
```
Comment (nataliemisasi) [inline help/en/docs/widget-custom.md:579]:
```suggestion
A comma will not work to separate the links. 
```

File: help/en/docs/widget-custom.md [hunk 4]
```diff
@@ -572,6 +572,18 @@ To copy the URL for an image, right click on the photo then 'Copy image address'
...
 
+Additionally, you can add multiple images for a specific entry. First, head to the 🌳 All National Park Sites Card and click into the Park Image field. 
+
+*![image-viewer-park-card](images/widget-custom/image-viewer-park-card.png)*
+
+Here, you can paste an additional link to add another photo. For proper spacing, use a space or a new line. A comma will not work to seperate the links. 
+
+*![image-viewer-multiple-links](images/widget-custom/image-viewer-multiple-links.png)*
+
+Now, there are two photos that you can toggle through by clicking on the arrows in the 🌳 All National Park Sites Custom Widget.
```
Comment (nataliemisasi) [inline help/en/docs/widget-custom.md:583]:
```suggestion
When multiple image URLs are present, the image viewer custom widget will function as a carousel. Click the arrows to view additional images.
```

**Review by nataliemisasi — COMMENTED** [review]
Made some suggestions to simplify and generalize

**Review by nbush — APPROVED** [review]
I don't like encouraging hotlinking, but otherwise the content looks good.

**Pull request merged**